app/routers/guest/meta.py

```diff
@@ -0,0 +1,37 @@
+from typing import Any
+
+from fastapi import APIRouter, HTTPException, status
+
+from app import models
+from app.i18n import _
+
+router = APIRouter()
+
+
+@router.get(
+    "/meta/{domain}",
+    tags=["meta"],
+)
+async def get_settings_by_domain(domain: str) -> list[dict[str, str | Any]]:
+    """
+    Get the keys and localized descriptions of a specific domain, where a domain can be:
+        - BorrowerType
+        - CreditType
+        - BorrowerDocumentType
+        - BorrowerSize
+        - BorrowerSector,
```

Comment:
reStructured Text sometimes gets upset about missing newlines. Sort.

```suggestion

        - BorrowerDocumentType
        - BorrowerSector
        - BorrowerSize
        - BorrowerType
        - CreditType
```